recipes/gsalign/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+make Compiler=$CXX CXX=$CXX CC=$CC LDFLAGS="${LDFLAGS}" CFLAGS="${CFLAGS} -msse4.2 -mpopcnt -fPIC" CXXFLAGS="${CXXFLAGS} -msse4.2 -mpopcnt -fPIC"
+cp bin/GSAlign bin/bwt_index $PREFIX/bin
```

Comment:
If `bin/bwt_index` is the same file as in gh-18694 and gh-18702, you might want to consider putting that into a separate package.
Depending on the user's `conda config` settings, having file with the same path in multiple packages will either prevent those from being installed simultaneously or lead to unexpected errors:
```
> conda config --describe path_conflict
# # path_conflict (PathConflict)
# #   The method by which conda handle's conflicting/overlapping paths
# #   during a create, install, or update operation. The value must be one
# #   of 'clobber', 'warn', or 'prevent'. The '--clobber' command-line flag
# #   or clobber configuration parameter overrides path_conflict set to
# #   'prevent'.
# # 
# path_conflict: clobber
```
The default setting is `clobber` which means `conda` will just install all packages as instructed, but will also not pay attention to the conflicting file when removing any package. Imagine the following:
```
> # install the two packages, gsalign and kart, that both contain bin/bwt_index
> conda install gsalign kart
> # remove only one of the packages
> conda remove kart
> # now, bin/bwt_index has been removed alongside all of kart
> bwt_index
bash: bwt_index: command not found
> # ergo: the currently still installed package gsalign is incomplete/broken
```

You can avoid this conflict easily if you put `bin/bwt_index` in a separate package and add that package as a `run` dependencies to the other packages.